File: ch4/exo1.py
#!/usr/bin/env python3

## print the corresponding family name given a user name

family_name = {'fred': 'flinstone',
            'barney': 'rubble',
            'wilma': 'flinstone'
            }

name = input('Enter a name: ')

if name in family_name:
    print('Name is: {}'.format(family_name[name]))
else:
    print('Name not found.')

File: ch4/exo1.pl
#!/usr/bin/env perl
use strict;
use warnings;
use utf8;

## print the corresponding family name given a user name

my %family_name = (
	fred 	=> 'flinstone',
	barney 	=> 'rubble',
	wilma	=> 'flinstone',
);

print("Enter a name: ");

chomp(my $name = <STDIN>);

if (exists $family_name{$name}) {
	print("Name found: $family_name{$name}\n");
}
else {
	print("Name not found in database.\n");
}
